code/modules/clothing/clothing.dm

```diff
@@ -159,6 +152,89 @@
 		how_cool_are_your_threads += "</span>"
 		. += how_cool_are_your_threads.Join()
 
+	if(LAZYLEN(armor_list))
+		armor_list.Cut()
+	if(armor.bio)
+		armor_list += list("TOXIN" = armor.bio)
+	if(armor.bomb)
+		armor_list += list("EXPLOSIVE" = armor.bomb)
+	if(armor.bullet)
+		armor_list += list("BULLET" = armor.bullet)
+	if(armor.energy)
+		armor_list += list("ENERGY" = armor.energy)
+	if(armor.laser)
+		armor_list += list("LASER" = armor.laser)
+	if(armor.magic)
+		armor_list += list("MAGIC" = armor.magic)
+	if(armor.melee)
+		armor_list += list("MELEE" = armor.melee)
+	if(armor.rad)
+		armor_list += list("RADIATION" = armor.rad)
+	if(armor.stamina)
+		armor_list += list("STAMINA" = armor.stamina)
+
+	if(LAZYLEN(durability_list))
+		durability_list.Cut()
+	if(armor.fire)
+		durability_list += list("FIRE" = armor.fire)
+	if(armor.acid)
+		durability_list += list("ACID" = armor.acid)
+
+	if(LAZYLEN(armor_list) || LAZYLEN(durability_list))
+		. += "<span class='notice'>It has a <a href='?src=[REF(src)];list_armor=1'>tag</a> listing its protection classes.</span>"
```

Comment:
This proabably doesn't need to be done every single time on examine

Reply:
how else would it gather that the lists have enough length to tell you that the object has a tag on it